backend/windmill-common/src/more_serde.rs

```diff
@@ -33,3 +35,32 @@ pub fn default_id() -> String {
 pub fn is_default<T: Default + std::cmp::PartialEq>(t: &T) -> bool {
     &T::default() == t
 }
+
+pub fn maybe_number_opt<'de, T, D>(deserializer: D) -> Result<Option<T>, D::Error>
+where
+    D: Deserializer<'de>,
+    T: FromStr + serde::Deserialize<'de>,
+    <T as FromStr>::Err: Display,
+{
+    #[derive(Deserialize)]
+    #[serde(untagged)]
+    enum NumericOrNull<'a, T> {
+        String(String),
+        Str(&'a str),
+        FromStr(T),
```

Comment:
Not sure to follow why this is called `FromStr`, shouldn't it be called `Raw` or equivalent ?

Reply:
I'm actually new to rust, but that was the best thing I got to work. If you got anything better, feel free to add

Reply:
It's very good code, I was just wondering about the naming of this enum case. It seems that this branch represent the raw value and could be named as such